rails-task-list/config/routes.rb

```diff
@@ -1,11 +1,15 @@
 Rails.application.routes.draw do
 
   root 'tasks#index'
-  get  '/tasks'       => 'tasks#index'
-  get  '/tasks/new'   => 'tasks#new'
-  get  '/tasks/:id'   => 'tasks#show'
-  post '/tasks/'      => 'tasks#create'
-  delete '/tasks/:id' => 'tasks#destroy'
+  get  '/tasks'               => 'tasks#index'
+  get  '/tasks/new'           => 'tasks#new'
+  get  '/tasks/:id/edit'      => 'tasks#edit', as: :task
```

Comment:
Later this week we will be learning about "named routes" (what you get when you use the `as: :name` bit from this line), which are very powerful and convenient. However, until we have a chance to talk about exactly how they work it might be best to avoid using them.